server/controllers/v1/tests/post.test.ts

```diff
@@ -71,6 +74,86 @@ describe('Test v1 - Post Controller', () => {
       expect(result.status).toBe(200);
     });
 
+    it('should allow creating new tags when making a post', async () => {
+      const author = await makeValidUser();
+      const tags = ['csc108', 'bananaPepper'];
+
+      const result = await postController.createPost(
+        author.id,
+        'This is a new post!',
+        'This is a new post!This is a new post!',
+        'location',
+        10,
+        tags
+      );
+
+      if (result.data.result) {
+        const postTags = await getPostTags(result.data.result.id);
+        expect(postTags?.map((t) => t.text).sort()).toEqual(tags.sort());
```

Comment:
There should be a `toEqualSameMembers` matcher or something. Sorting is ignored.

Reply:
Just tried it and that didn't work... there doesn't seem to be a checker to ignore ordering, I checked jest docs and tried a couple other functions and none of them worked. Stackoverflow says to just use sort()